crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -105,6 +106,8 @@ where
             .archetype_component_access
             .extend(&self.archetype_component_access);
     }
+
+    fn default_config() {}
```

Comment:
If clippy lets you, I'd consider making this
```rust
fn default_config() ->() {}
```
This is also fine however, the only confusion arises due to exactly reviewing this PR in the diff viewer 😅

Reply:
Clippy doesn't like that...